circuitpy/lib/farm_ng/utils/main_loop.py

```diff
@@ -7,20 +19,17 @@
 
 from canio import BusState
 from canio import Message
-from supervisor import ticks_ms
 
 from .can import setup_can_default
-from .general import DtTracker
-from .general import TickRepeater
+from .debug_packet import FarmngDebugMemory
+from .debug_packet import FarmngDebugTimer
 from .nvm import nvm_serial_number
-from .packet import FarmngDebugMemory
-from .packet import FarmngDebugTimer
 from .packet import FarmngHeartbeat
 from .packet import NodeState
-from .packet import SupervisorReq
+from .ticks import DtTracker
```

Comment:
Let's remove the over complication for this being able to run the display and go to the App-based approach we now use